app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.plaid.linksample">
 
+  <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
This doesn't actually request the permissions, it just allowlists our test app to gain access to the camera so it also supports testing identity verification.